dora/core/client/fs/src/test/java/alluxio/client/file/dora/WorkerLocationPolicyTest.java

```diff
@@ -94,52 +94,62 @@ public void concurrentInitialization() {
   }
 
   @Test
-  // todo(bowen): this test can be flaky if the test subject is not thread safe
+  // Notes on thread safety:
+  // This test tries to ensure the test subject live up to its thread safety guarantees.
+  // 1. When the test subject is actually correctly implemented, this test will *always* pass.
+  // 2. When it is not, this test cannot reliably detect that, since it's a matter of
+  //    chance that race conditions manifest themselves. This test may appear to be flaky,
+  //    but chances are that there are thread safety issues with the test subject.
+  // To decrease the chance of false negatives, you can run this test manually multiple times
```

Comment:
nit: Add a word that "In this test we run concurrent operations for 100 times to reduce the chance of a false negative."